spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestChangelogTable.java

```diff
@@ -260,6 +260,14 @@ public void testManifestRewritesAreIgnored() {
         sql("SELECT id, _change_type FROM %s.changes ORDER BY id", tableName));
   }
 
+  @Test
+  public void testMetadataColumns() {
```

Comment:
Can we add test for all metadata column types?

Reply:
Added in the new commit.